tests/TestCase/ORM/MarshallerTest.php

```diff
@@ -2629,8 +2675,8 @@ public function testMergeAssociationWithfieldList()
         ];
         $marshall = new Marshaller($this->articles);
         $marshall->merge($entity, $data, [
-            'fieldList' => ['something'],
-            'associated' => ['Users' => ['fieldList' => ['extra']]]
+            $fields => ['something'],
```

Comment:
Could we keep the old and new test alongside and mark the old tests one deprecated (with version)?